server/src/definition/ModelDefinitionProvider.ts

```diff
@@ -51,28 +52,29 @@ export class ModelDefinitionProvider implements DbtNodeDefinitionProvider {
           document.positionAt(wordAbsoluteOffset + matches[1].index + 1),
           document.positionAt(wordAbsoluteOffset + matches[1].index + matches[1].text.length - 1),
         );
+
+        if (positionInRange(position, packageSelectionRange)) {
+          return this.searchPackageDefinition(dbtPackage, this.dbtRepository.models, packageSelectionRange);
+        }
       } else {
-        dbtPackage = `'${packageName}'`;
-        model = matches[0].text;
+        model = matches[0].text.slice(1, -1);
         packageSelectionRange = undefined;
         modelSelectionRange = Range.create(
```

Comment:
Can we extract method that will create SelectionRange from wordAbsoluteOffset and match? We do it 3 times now.